packages/messaging-api-line/src/LineTypes.ts

```diff
@@ -429,3 +429,172 @@ export type TextMessage = {
+   */
+  status: 'ready' | 'unready' | 'out_of_service';
+};
+
+export type NumberOfMessageDeliveries = InsightStatisticsResponse & {
+  /**
+   * Number of push messages sent to **all** of this LINE official account's friends (broadcast messages).
+   */
+  broadcast: number;
+  /**
+   * Number of push messages sent to **some** of this LINE official account's friends, based on specific attributes (targeted/segmented messages).
+   */
+  targeting: number;
+  /**
+   * Number of auto-response messages sent.
+   */
+  autoResponse: number;
+  /**
+   * Number of greeting messages sent.
+   */
+  welcomeResponse: number;
+  /**
+   * Number of messages sent from LINE Official Account Manager [Chat screen](https://www.linebiz.com/jp-en/manual/OfficialAccountManager/chats/screens/).
+   */
+  chat: number;
+  /**
+   * Number of broadcast messages sent with the [Send broadcast message](https://developers.line.biz/en/reference/messaging-api/#send-broadcast-message) Messaging API operation.
+   */
+  apiBroadcast: number;
+  /**
+   * Number of push messages sent with the [Send push message](https://developers.line.biz/en/reference/messaging-api/#send-push-message) Messaging API operation.
+   */
+  apiPush: number;
+  /**
+   * Number of multicast messages sent with the [Send multicast message](https://developers.line.biz/en/reference/messaging-api/#send-multicast-message) Messaging API operation.
+   */
+  apiMulticast: number;
+  /**
+   * Number of replies sent with the [Send reply message](https://developers.line.biz/en/reference/messaging-api/#send-reply-message) Messaging API operation.
+   */
+  apiReply: number;
+};
+
+export type NumberOfFollowers = InsightStatisticsResponse & {
+  /**
+   * The number of times, as of the specified `date`, that a user added this LINE official account as a friend. The number doesn't decrease when a user blocks the account after adding it, or when they delete their own account.
+   */
+  followers: number;
+  /**
+   * The number of users, as of the specified `date`, that the official account can reach with messages targeted by gender, age, or area. This number includes users for whom we estimated demographic attributes based on their activity in LINE and LINE-connected services.
+   */
+  targetedReaches: number;
+  /**
+   * The number of users blocking the account as of the specified `date`. The number decreases when a user unblocks the account.
+   */
+  blocks: number;
+};
+
+export type NumberOfMessageDeliveriesResponse =
+  | InsightStatisticsResponse
+  | NumberOfMessageDeliveries;
+
+export type NumberOfFollowersResponse =
+  | InsightStatisticsResponse
+  | NumberOfFollowers;
+
+type PercentageAble = {
+  percentage: number;
+};
+
+export type FriendDemoGraphics = {
```

Comment:
should be `FriendDemographics`

Reply:
https://github.com/line/line-bot-sdk-nodejs/blob/000f681b782476dbdc46d81b842d715f64617d5a/lib/types.ts#L2025

I'd like to use the same type with line sdk, so our users may migrate from there.

Reply:
@tw0517tw You can send a PR to there and see how it goes.

Reply:
I've sent one https://github.com/line/line-bot-sdk-nodejs/pull/177

Reply:
done.